src/BlazorWebView/src/SharedSource/WebView2WebViewManager.cs

```diff
@@ -125,7 +127,7 @@ public class WebView2WebViewManager : WebViewManager
 		/// <param name="jsComponents">Describes configuration for adding, removing, and updating root components from JavaScript code.</param>
 		/// <param name="hostPageRelativePath">Path to the host page within the <paramref name="fileProvider"/>.</param>
 		/// <param name="blazorWebViewHandler">The <see cref="BlazorWebViewHandler" />.</param>
-		public WebView2WebViewManager(
```

Comment:
I made the two constructors internal because I'm not aware of a reason for them to be public. It's used across all the supported platforms via shared source.

If I'm wrong, I'm happy to change this back. However we've changed the parameter list several times now, so it suggests we'll keep changing it in the future, and maintaining back-compat if it's a public API could become very messy.